platform/linuxbsd/x11/gl_manager_x11.h

```diff
@@ -74,17 +74,17 @@ class GLManager_X11 {
 	};
 
 	struct GLDisplay {
-		GLDisplay() { context = nullptr; }
+		GLDisplay();
 		~GLDisplay();
 		GLManager_X11_Private *context = nullptr;
-		::Display *x11_display;
-		XVisualInfo x_vi;
+		::Display *x11_display = nullptr;
+		XVisualInfo x_vi = {};
```

Comment:
That's unrelated to the changes in this PR, but since it triggered rebuilding the engine from scratch, it caught new warnings which were not emitted when reusing the cached objects. If this works to fix the warnings, I'll split it off to a separate PR.

Reply:
Moved to #80615, I'm including this commit in this PR for now until it's merged.